evap/staff/tests/test_views.py

```diff
@@ -1722,6 +1724,25 @@ def test_import_valid_participants_file(self):
         page = self.app.get(self.url, user=self.manager)
         self.assertNotContains(page, 'Import previously uploaded file')
 
+    def test_replace_valid_participants_file(self):
+        page = self.app.get(self.url2, user=self.manager)
+
+        original_participant_count = self.evaluation2.participants.count()
+
+        form = page.forms["participant-import-form"]
+        form["excel_file"] = (self.filename_valid,)
+        page = form.submit(name="operation", value="test-participants")
+
+        self.assertContains(page, 'Import previously uploaded file')
+        self.assertEqual(self.evaluation2.participants.count(), original_participant_count)
```

Comment:
In theory, we should split up testing the test run, which is the same for import and replace, and then testing the actual import and replace runs (so these assertions here are not duplicated) (same for the contributors tests)